infoschema/tables_test.go

```diff
@@ -743,76 +740,6 @@ func (s *testTableSuite) TestForTableTiFlashReplica(c *C) {
 	tk.MustQuery("select TABLE_SCHEMA,TABLE_NAME,REPLICA_COUNT,LOCATION_LABELS,AVAILABLE, PROGRESS from information_schema.tiflash_replica").Check(testkit.Rows("test t 2 a,b 1 1"))
 }
 
-func (s *testClusterTableSuite) TestForClusterServerInfo(c *C) {
```

Comment:
Why remove the test cases?

Reply:
By mistake, sorry!  Just wondering why this unit test failed when I run `go test` under this package `infoshema`.